frontend/src/containers/LibraryPage/LibraryPage.js

```diff
@@ -107,67 +137,106 @@ class LibraryPage extends React.Component {
       }
     };
 
+    const renderDownloadSnackbars = () => {
+      if (showDownloadError) {
+        return (
+          <CustomSnackbar
+            message="Ooops.. Something wrong happened. Please try again later"
+            onClose={() => this.setState({ showDownloadError: false })}
+            severity="error"
+            topCenter
```

Comment:
I did bottom left on my branch, so I don't know which variant we should use

Reply:
I prefer the top one. It's clearly visible